lib/src/numeric/constants.dart

```diff
@@ -0,0 +1,167 @@
+/// A collection of frequently used mathematical constants.
+class Constants {
+  /// The number e
+  static const double e = 2.7182818284590452353602874713526624977572470937000;
```

Comment:
This (and some below) are already in `dart:math`.

Reply:
I removed the file